src/test/java/org/springframework/cloud/bindings/boot/EurekaBindingsPropertiesProcessorTest.java

```diff
@@ -16,48 +16,180 @@
 
 package org.springframework.cloud.bindings.boot;
 
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
 import org.springframework.cloud.bindings.Binding;
 import org.springframework.cloud.bindings.Bindings;
 import org.springframework.cloud.bindings.FluentMap;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.FileSystemResource;
 import org.springframework.mock.env.MockEnvironment;
 
+import java.io.File;
 import java.nio.file.Paths;
 import java.util.HashMap;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.springframework.cloud.bindings.boot.EurekaBindingsPropertiesProcessor.TYPE;
+import static org.springframework.cloud.bindings.boot.TestHelper.resourceAsString;
 
 @DisplayName("Eureka BindingsPropertiesProcessor")
 final class EurekaBindingsPropertiesProcessorTest {
-
-    private final Bindings bindings = new Bindings(
+    private Bindings bindings = new Bindings(
             new Binding("test-name", Paths.get("test-path"),
                     new FluentMap()
                             .withEntry(Binding.TYPE, TYPE)
                             .withEntry("uri", "test-uri")
-                            .withEntry("client-id", "test-client-id")
```

Comment:
why dont you keep them and test them in `    @DisplayName("contributes only base properties when no auth is set")` ?

Reply:
There is a valid test case where the binding contains the URI, but no credentials. We wanted to test that case. The case where oauth2 credentials are set is in a separate, more explicit test now.

Reply:
> The case where oauth2 credentials are set is in a separate, more explicit test now.

true, I did not notice; in `@DisplayName("contributes oauth properties when set")`

All good then!